package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "cohere-ai",
-  "version": "1.0.0",
+  "version": "1.0.6",
```

Comment:
version in package.json is 1.0.7, but this is 1.0.6

Reply:
Good catch, but that's fine as is, the package-lock.json can be ignored as it auto generated and only consumed by npm behind the scenes. It's not revelant to anyone using the sdk.